draft-ietf-quic-transport.md

```diff
@@ -5078,6 +5086,7 @@ Preferred Address {
   IPv4 Port (16),
   IPv6 Address (128),
   IPv6 Port (16),
+  Disable Migration (8),
```

Comment:
Let's phrase this positively, no matter which way we resolve the per-family question.

Reply:
I think my not-so-strong preference would be to call this value "disable_active_migration" so that it would be consistent with the Transport Parameter that we use to advertise the property of the original address.

Reply:
That was my intent.  I don't really care which way it's pointed, but pointing the TP and the TP-field in opposite directions is needlessly confusing.

Reply:
Agreed, it would be nice to have them be in the same direction (and especially the name "Disable Active Migration" since they have the same meaning)